mettle/src/crypttlv.c

```diff
@@ -0,0 +1,205 @@
+#include "crypttlv.h"
+
+size_t aes_decrypt(struct tlv_encryption_ctx* ctx, const unsigned char* data, size_t data_len, unsigned char* result)
+{
+#ifndef __MINGW32__
```

Comment:
This isn't really the right check, but I suppose I probably must have used this elsewhere too.